package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@gofynd/fdk-cli",
-  "version": "5.0.2",
+  "version": "5.0.3",
```

Comment:
version is not updated in package-lock.json @bhargavprajapatiFynd

Reply:
Use `npm version patch` command

Reply:
done